content/en/containers/cluster_agent/clusterchecks.md

```diff
@@ -245,6 +265,28 @@ clusterAgent:
           name: "<EXAMPLE_NAME>"
 ```
 
+{{% /tab %}}
+{{% tab "Operator" %}}
+Use the `spec.override.clusterAgent.extraConfd.configDataMap` field to define your check configuration:
+
+```yaml
+spec:
+#(...)
+  override:
+    clusterAgent:
+      extraConfd:
+        configDataMap:
+          <INTEGRATION_NAME>.yaml: |-
```

Comment:
I think the `<INTEGRATION_NAME>.yaml` could be filled in with `http_check.yaml` for this section and the Helm tab to match the instructions we give in the DaemonSet tab about adding a `http_check.yaml` file to the Cluster Agent